service/models/payment_method.py

```diff
@@ -164,3 +165,8 @@ def find_by_user_id(cls, user_id, q=None):
         if q is None:
             q = cls.query
         return q.filter(cls.user_id == user_id)
+
+    @classmethod
+    def find_default_for_user(cls, user_id):
```

Comment:
this is not used anywhere but in the test. maybe you can move the logic from the route to this specific method, and call it something like `set_default_for_user` ?